CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add getter for path field in NestedQueryBuilder ([#4636](https://github.com/opensearch-project/OpenSearch/pull/4636))
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add support for ppc64le architecture ([#5459](https://github.com/opensearch-project/OpenSearch/pull/5459))
+- [Searchable Snapshots] Add integration test to validate cache files are closed after use ([#6215](https://github.com/opensearch-project/OpenSearch/pull/6215)
```

Comment:
[Test-only changes](https://github.com/opensearch-project/OpenSearch/blob/90fefa262c7089eefe846e49c3acb96469d98eb6/CONTRIBUTING.md?plain=1#L134) can be omitted from the changelog.

Also, though this is moot since we don't need an entry for this change, but any change that will be backported to 2.x should _only_ be in the `[Unreleased 2.x]` section because this will never be "unreleased" relative to 3.0 because it will go into a 2.x release.

Reply:
@andrross Thanks for your explanation and pointing me to the document of the changelog! The guide is indeed very clear.